components/NavBar.tsx

```diff
@@ -104,6 +105,11 @@ export default function NavBar(props: Props) {
                     <><ExitIcon/>&nbsp;Sign out</>
                   }
                 </MenuItem>
+                <Link to='/apps/my-waggles'>
```

Comment:
Looking good!    @SammiH2022 I think we we should (eventually):

- update this route to `/my-devices` or `/my-waggle-devices` (so that it's not under `/apps`).  Or, `/devices/mine` would be ok too.
- use clearer icon?